cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -311,6 +311,14 @@ typedef enum {
     I2C_SPEED_HIGH      = 3400000U,    /**< high speed mode:   ~3.4Mbit/s */
 } i2c_speed_t;
 /** @} */
+
+static inline gpio_t i2c_pin_sda(i2c_t dev) {
+    return i2c_config[dev].sda_pin;
+}
+
+static inline gpio_t i2c_pin_scl(i2c_t dev) {
+    return i2c_config[dev].scl_pin;
+}
```

Comment:
`i2c_config` isn't know at this point.
The `board.h` includes `periph_cpu.h` (which then includes `periph_cpu_common.h`) before `i2c_config` is defined.

This could be fixed by a `#define i2c_pin_sda(dev) i2c_config[dev].sda_pin`. But I don't know if this could be called a *clean solution*.